config/peerpods/podvm/bootc/Containerfile.rhel

```diff
@@ -39,6 +39,42 @@ COPY --from=payload /podvm-binaries.tar.gz /podvm-binaries.tar.gz
 RUN tar -xzvf podvm-binaries.tar.gz -C /
 RUN sed -i 's#What=/kata-containers#What=/var/kata-containers#g' /etc/systemd/system/run-kata\\x2dcontainers.mount
 
+
+
+########## Nvidia podVM target ##########
```

Comment:
operator's built podvm nvidia gpu is optional and protected by `ENABLE_NVIDIA_GPU=no` by default. Wouldn't make sense to have it optional here too?

Reply:
ah, got it. Makes sense.